go.mod

```diff
@@ -19,6 +19,8 @@ require (
 	github.com/cockroachdb/logtags v0.0.0-20230118201751-21c54148d20b // indirect
 	github.com/cockroachdb/pebble v0.0.0-20230209160836-829675f94811 // indirect
 	github.com/cockroachdb/redact v1.1.3 // indirect
+	github.com/consensys/bavard v0.1.13 // indirect
+	github.com/consensys/gnark-crypto v0.9.1-0.20230105202408-1a7a29904a7c // indirect
```

Comment:
Why is this using a different version of gnark-crypto than go-boost-utils? It should be `v0.7.1`.

Reply:
Ah, it's because Geth [uses gnark-crypto](https://github.com/ethereum/go-ethereum/blob/master/go.mod#L16) for two of their fuzzers:

* [`bn256_fuzz.go`](https://github.com/ethereum/go-ethereum/blob/e6b6a8b738069ad0579f6798ee59fde93ed13b43/tests/fuzzers/bn256/bn256_fuzz.go#L28)
* [`bls12381_fuzz.go`](https://github.com/ethereum/go-ethereum/blob/e6b6a8b738069ad0579f6798ee59fde93ed13b43/tests/fuzzers/bls12381/bls12381_fuzz.go#L29)

And because this is just for fuzzers, it seems perfectly fine to use the latest version.

I'm fairly certain that Go is smart enough to handle two version of indirect dependencies.

Reply:
Hmm actually, I think that mev-boost is using `v0.9.1`, not `v0.7.1` that go-boost-utils requires. If true, that's pretty disappointing and feels like a Go problem. If you delete the module cache (`go clean -modcache`) and rebuild, it only downloads `v0.9.1`. And I just realized that go-boost-utils works with both versions, so the tests aren't an indicator.

Reply:
Fixed with a `replace` in the `go.mod` file.